packages/core/src/app/address/StaticAddress.tsx

```diff
@@ -33,12 +33,21 @@ export interface StaticAddressEditableProps extends StaticAddressProps {
 interface WithCheckoutStaticAddressProps {
     countries?: Country[];
     fields?: FormField[];
+    loadAddressFields: () => Promise<CheckoutSelectors>
 }
 
 const StaticAddress: FunctionComponent<
     StaticAddressEditableProps & WithCheckoutStaticAddressProps
-> = ({ countries, fields, address: addressWithoutLocalization }) => {
+> = ({ countries, fields, loadAddressFields, address: addressWithoutLocalization }) => {
     const { isPayPalAxoEnabled, paypalConnectAddresses } = usePayPalConnectAddress();
+
+    useEffect(() => {
+        const fetchData = async () => {
+            await loadAddressFields();
+        }
+        if (!countries) fetchData();
```

Comment:
Can you please write this as 
```suggestion
        if (!countries) {
            fetchData()
        };
```